docs/configuration/interfaces/ethernet.rst

```diff
@@ -295,5 +303,3 @@ Operation
         BR margin, min          : 0%
         Vendor SN               : FNS092xxxxx
         Date code               : 0506xx
-
-.. stop_vyoslinter
```

Comment:
Linter is started in Line 230 - why remove it here?

Reply:
this is fine, linter always run and must start after stop.